docker/windows/Dockerfile

```diff
@@ -1,3 +1,5 @@
+# hadolint shell = cmd.exe
+#
```

Comment:
Is this needed?

Reply:
Unfortunately this shell pragma should be used when one wants to lint a Dockfile dedicated to Windows.
Otherwise you will end up with false positives from ShellCheck, see [hadolint shell pragma](https://github.com/hadolint/hadolint/pull/708)

Reply:
Okay - np!